test/util/page-number-functions.spec.js

```diff
@@ -0,0 +1,77 @@
+const { expect } = require('chai')
+const { findPageNumbers, findFirstPage, removePageNumber } = require('../../lib/util/page-number-functions')
+
+describe('functions: findPageNumbers', () => {
+  it('Search, coerce and store page number', () => {
+    const array = [{ str: '1' }, { str: 'test' }, { str: 'how' }, { str: 'to' }, { str: 'find' }, { str: 'page' }, { str: '3' }, { str: 'number' }]
+    expect(findPageNumbers({}, 3, array)).to.eql({ 3: [1, 3] })
+  })
+})
+
+describe('functions: findFirstPage', () => {
+  it('Return first page index and number with completed Hashtable', () => {
```

Comment:
I realise the other test spec doesn't do this (so I will correct it later).. `it()` invocations should be such that the string argument forms a proper sentence, ie..
```suggestion
  it('returns first page index and number with completed Hashtable', () => {
```

Reply:
Ok I will change my `it()` so that the string forms a proper sentence